tests/integration-tests/configs/common/common.yaml

```diff
@@ -776,3 +776,10 @@ health_checks:
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
         oss: ["alinux2"]
         schedulers: ["slurm"]
+schedulers:
```

Comment:
Watch out! The `schedulers` section is already defined a bit above. Please just append your test in that definition, rather than re-defining `schedulers`.

Reply:
Thanks for your reminding Jacopo, I didn't notice that.